.gitignore

```diff
@@ -21,7 +21,7 @@ GOPOWERSTORE_TEST.env
 
 # IDE
 .idea
-
+.vscode
```

Comment:
do we need to commit this file?

Reply:
ya, otherwise it won't ignore .vscode folder like the way we did for .idea

Reply:
ya, otherwise it won't ignore .vscode folder like the way we did for .idea